### ISSUE TITLE
serenity-jira-requirements-provider not running in Gradle

### ISSUE THREAD

**Issue by tokeh** [serenity-bdd/serenity-jira#62]
Hi,
I'm currentliy trying to get the "feature coverage" of my JIRA project. I'm using gradle and added the serenity-jira-requirements-provider, serenity-jira-plugin dependencies to my build.gradle. The test and aggregate tasks are running without any error messages but epics and stories aren't added to my report.
Even falsifying my serenity.properties file on purpose didn't lead to any errors. I'm assuming that the requirements provider isnt't running at all. So my question is what am I doing wrong? What do I have to add/change in my build.gradle to get the requirements provider running? Do you have any working examples to copy from?

Best regards
Tobias

**Comment by cliviu** [comment]
Hi @tokeh . Can you please post your build.gradle ?

**Comment by tokeh** [comment]
Yes of course :)

```gradle
group 'BDD_Tutorial'
version '1.0-SNAPSHOT'

apply plugin: 'java'
apply plugin: 'idea'
apply plugin: 'net.serenity-bdd.aggregator'

sourceCompatibility = 1.8

def serenity_version = "1.1.42"

repositories {
  mavenCentral()
  jcenter()
}

buildscript {
  repositories {
    mavenCentral()
    jcenter()
  }
  dependencies {
    classpath("net.serenity-bdd:serenity-gradle-plugin:1.1.42")
    classpath("net.serenity-bdd:serenity-jira-plugin:1.1.1")
    classpath("net.serenity-bdd:serenity-jira-requirements-provider:1.1.1")
  }
}

dependencies {
  compile group: 'junit', name: 'junit', version: '4.12'
  compile group: 'org.assertj', name: 'assertj-core', version: '3.5.2'
  compile group: 'org.slf4j', name: 'slf4j-simple', version: '1.7.21'
  compile group: 'org.hamcrest', name: 'hamcrest-all', version: '1.3'

  compile group: 'info.cukes', name: 'cucumber-core', version: '1.2.5'
  compile group: 'info.cukes', name: 'cucumber-java', version: '1.2.5'
  compile group: 'info.cukes', name: 'cucumber-junit', version: '1.2.5'
  compile group: 'info.cukes', name: 'cucumber-jruby', version: '1.2.5'

  compile group: 'net.serenity-bdd', name: 'serenity-core', version: serenity_version
  compile group: 'net.serenity-bdd', name: 'serenity-junit', version: serenity_version
  compile group: 'net.serenity-bdd', name: 'serenity-cucumber', version: '1.1.11'
  compile group: 'net.serenity-bdd', name: 'serenity-gradle-plugin', version: serenity_version
  compile group: 'net.serenity-bdd', name: 'serenity-jira-plugin', version: '1.1.1'
  compile group: 'net.serenity-bdd', name: 'serenity-jira-requirements-provider', version: '1.1.1'
  compile group: 'net.serenity-bdd', name: 'serenity-jira-customfield-requirements-provider', version: '1.1.1'
}
```

**Comment by cliviu** [comment]
please try with serenity-core 1.2.1-rc.6 and serenity-jira-plugin 1.1.3-rc.1

**Comment by tokeh** [comment]
Unfortunately it's still not working :/
Just to be sure, the requirements-provider should run during the aggregate task, shouldn't it?
And is there any way to see some kind of Serenity logging?

**Comment by cliviu** [comment]
Hi @tokeh , sorry please try with serenity-core 1.2.1 and  serenity-jira-plugin/requirements-provider 1.1.3-rc.2

**Comment by cliviu** [comment]
yes, logging is possible but you have to configure some logging framework to work with sl4j (see http://www.slf4j.org/)

**Comment by tokeh** [comment]
Hi @cliviu, I tried using serenity-core 1.2.1 and serenity-jira-plugin/requirements-provider 1.1.3-rc.2 but now the aggregate task isn't working correctly anymore. The report is getting generated but the links to "Capabilities" and "Features" are broken. And the connection to JIRA is still not working, too :/

**Comment by cliviu** [comment]
Hi, how do you mean that the connection to jira is not working ? Do you have a jira.project set in your serenity properties ?

**Comment by tokeh** [comment]
Hey, when Serenity is checking the feature coverage it has to sign into JIRA and get all the necessary data. That's what I mean with "connection to JIRA". I have a serenity properties file with jira.project, url, username and password defined. But when I'm changing the data nothing happens (no error, etc.)  that's why I'm assuming the requirements-provider task isn't running at all. 

Cheers and happy holidays :)

**Comment by cliviu** [comment]
hi , Do you have any @issue:XXX-XX annotation in your .feature files ?  What happens when trying to follow the generated link in the report ? If there is no @issue/@issues tag in the project, no connection to the jira server will be done so that is maybe why you do not get any exceptions ... 
 Thanks and happy holidays to you too  :-) !

**Comment by michaelgx** [comment]
Having this problem aswell.

**Comment by michaelgx** [comment]
[log1.txt](https://github.com/serenity-bdd/serenity-jira/files/748247/log1.txt)
Here is the log and here is the serenity.properties file:
webdriver.base.url=https://business.aib.ie/businesslogin
webdriver.phantomjs.driver=/usr/bin
phantomjs.binary.path=/usr/bin/phantomjs2
jira.url=https://jira.aib.pri
serenity.jira.workflow.active=true
serenity.skip.jira.updates=true
serenity.logging=VERBOSE
serenity.jira.workflow=workflow.groovy
serenity.outputDirectory=build/serenity
jira.project=DBBC
serenity.use.requirements.directories=false

**Comment by cliviu** [comment]
What is serenityVersion and serenityJiraVersion ?

**Comment by michaelgx** [comment]
In :

https://github.com/michaelgx/testing/blob/master/gradle.properties
